README.md

```diff
@@ -39,6 +40,27 @@ Unlike `vde_vmnet`, `socket_vmnet` does not depend on VDE.
 
 Requires macOS 10.15 or later.
 
+### From binary
+
+```bash
+VERSION="$(curl -fsSL https://api.github.com/repos/lima-vm/socket_vmnet/releases/latest | jq -r .tag_name)"
+FILE="socket_vmnet-${VERSION:1}-$(uname -m).tar.gz"
+
+# Download the binary archive
+curl -OSL "https://github.com/lima-vm/socket_vmnet/releases/download/${VERSION}/${FILE}"
+
+# (Optional) Attest the GitHub Artifact Attestation using GitHub's gh command (https://cli.github.com)
+gh attestation verify --owner=lima-vm "${FILE}"
+
+# (Optional) Preview the contents of the binary archive
+tar tzvf "${FILE}"
+
+# Install /opt/socket_vmnet from the binary archive
+sudo tar Cxzvf / "${FILE}" opt/socket_vmnet
```

Comment:
I think we need to change the tarball so it does not include /opt/socket_vmnet - this will make it eaiser and safer to install it in any location. This example installs only opt/socket_vmnet form the tarball. If you remove the argument you need to trust that the tarball does not contain other unexpected directories. I think that lima tarball does not have this issue, and you can just unpack it where you want.

Reply:
> I think that lima tarball does not have this issue, and you can just unpack it where you want.

The key difference is that unlike the lima tarball the socket_vmnet tarball is not relocatable due to the hard-coded binary path in the launchd plists